cassandra/tablets.py

```diff
@@ -0,0 +1,48 @@
+import bisect
+from threading import Lock
+
+# Experimental, this interface and use may change
+class Tablet(object):
+    """
+    Represents a single ScyllaDB tablet.
+    """
+    last_token = 0
+    new_replicas = None
+    replicas = None
+    stage = ""
+
+    def __init__(self, last_token = 0, new_replicas = None, replicas = None, stage = ""):
+        self.last_token = last_token
+        self.new_replicas = new_replicas
+        self.replicas = replicas
+        self.stage = stage
+
+    # Experimental, this interface and use may change
+    @staticmethod
+    def _is_valid_tablet(row):
+        return row.get("replicas") is not None and len(row.get("replicas")) != 0 and row.get("table_name") is not None and row.get("keyspace_name") is not None
+
+    # Experimental, this interface and use may change
+    @staticmethod
+    def from_row(row):
+        if Tablet._is_valid_tablet(row):
+            tablet = Tablet(row.get("last_token"), row.get("new_replicas"), row.get("replicas"), row.get("stage"))
+            return tablet
+        return None
+
+# Experimental, this interface and use may change
+class Tablets(object):
+    _tablets = {}
+    _tablets_lock = Lock()
+
+    def __init__(self, tablets):
+        self._tablets = tablets
+    
+    # Experimental, this interface and use may change
+    def get_tablet_for_key(self, keyspace, table, t):
+        tablet = self._tablets.get((keyspace, table), [])
+        if tablet == []:
+            return Tablet()
```

Comment:
I feel it would better to return `None` in this case instead of default value (which then caller need to recognize).

Reply:
done